server/src/main/java/org/opensearch/cluster/routing/allocation/AllocationService.java

```diff
@@ -608,9 +642,9 @@ private void applyStartedShards(RoutingAllocation routingAllocation, List<ShardR
                 : "shard started for unknown index (shard entry: " + startedShard + ")";
             assert startedShard == routingNodes.getByAllocationId(startedShard.shardId(), startedShard.allocationId().getId())
                 : "shard routing to start does not exist in routing table, expected: "
-                    + startedShard
-                    + " but was: "
-                    + routingNodes.getByAllocationId(startedShard.shardId(), startedShard.allocationId().getId());
+                + startedShard
+                + " but was: "
+                + routingNodes.getByAllocationId(startedShard.shardId(), startedShard.allocationId().getId());
```

Comment:
indentation change?

Reply:
Will resolve once compilation pass, by ./gradlew spotlessApply to have uniformity of identataiton